File: Boca/JAVA_2/Q5/src/DVD.java
public class DVD extends Produto {
    private int duracao;

    public DVD (int codigoDeBarras, String nome, double preco, int duracao) {
        super(codigoDeBarras, nome, preco);
        this.duracao = duracao;
    }

    public int getDuracao() {
        return duracao;
    }
    
}

File: Boca/CPP/07/DVD.cpp
#include "DVD.hpp"

DVD::DVD(int codigoBarras, const string& nome, double preco, int duracao) : Produto(codigoBarras, nome, preco), duracao(duracao) {}

void DVD::print(ostream &out) const {
    Produto::print(out);
    out << " - " << duracao << " minutos";
}

ostream& operator<< (ostream &out, DVD &dvd) {
    dvd.print(out);
    return out;
}

DVD::~DVD() {}